### ISSUE TITLE
Need for CONTRIBUTORS.md

### ISSUE THREAD

**Issue by trevor-henderson** [jackcmeyer/open-fantasy-football-api#2]
Address the Need for a `CONTRIBUTORS.md` file to establish a workflow such that other developers have context on how to contribute to the project and establish community guidelines.

**Comment by jackcmeyer** [comment]
@trevor-henderson definitely. will work on getting this setup. going to move this issue to https://github.com/jackcmeyer/open-fantasy-football since i'm going to go with a monorepo style for this project.
